WooCommerce/WooCommerceTests/ViewRelated/Products/SKU Scanner/BarcodeSKUScannerErrorNoticeFactoryTests.swift

```diff
@@ -0,0 +1,36 @@
+import Foundation
+import XCTest
+import Yosemite
+@testable import WooCommerce
+
+final class BarcodeSKUScannerErrorNoticeFactoryTests: XCTestCase {
+    func test_notice_when_a_generic_error_is_passed_then_returns_generic_notice() {
+        let error = NSError(domain: "disconnect", code: 134)
+        let result = BarcodeSKUScannerErrorNoticeFactory.notice(for: error, actionHandler: {})
+
+        XCTAssertEqual(result.title, NSLocalizedString("Cannot add Product to Order.", comment: ""))
```

Comment:
Since the title is of `String` type we do not seem to need the `NSLocalizedString` (or could it fail if there's some dev setting on a different language? 🤔 )

```suggestion
        XCTAssertEqual(result.title, "Cannot add Product to Order.")
```

Reply:
Yeah, In case the dev setting is changed to another language, I preferred to use `NSLocalizedString` here. You can see that we do the same in many cases throughout the tests classes. It also makes it explicit that we are using localized strings here.